File: Java/arrays/array_sum.java
package br.com.geekuniversity.secao08;

import java.util.Scanner;

public class Exercicio04 {

	public static void main(String[] args) {
		//Declaração
		int vetor[] = new int[20], soma = 0;
		Scanner entrada = new Scanner(System.in);
		
		//Entrada
		System.out.println("Digite 20 números inteiros: ");
		for (int i = 0; i < vetor.length; i++) {
			vetor[i] = entrada.nextInt();
			soma = soma + vetor[i];
		}
		
		//Saida
		System.out.println("Soma: " + soma);
		entrada.close();
	}
}

File: Python/arrays/array_sum.py
# coding=utf8
#Declaração
vetor = []
soma = 0

print("Digite vinte números inteiros para o vetor: ")
for i in range (0,20):
    i = int(input())
    vetor.append(i)
    soma = soma + i
    
print("Vetor: ", end = '')
for i in vetor:
    print(i, end = " ")
print("\nSoma: {0}". format(soma))